src/Hook/FeatureValue.php

```diff
@@ -36,8 +37,44 @@ class FeatureValue extends AbstractHook
         'actionFeatureValueDelete',
         'displayFeatureValueForm',
         'displayFeatureValuePostProcess',
+        'actionFeatureFormBuilderModifier',
+        'actionAfterCreateFeatureFormHandler',
+        'actionAfterUpdateFeatureFormHandler',
     ];
 
+    /**
+     * Hook for modifying feature form formBuilder
+     *
+     * @param array $params
+     *
+     * @throws PrestaShopDatabaseException
+     */
+    public function actionFeatureFormBuilderModifier(array $params)
+    {
+        $this->formModifier->modify($params['form_builder'], $this->dataProvider->getData($params));
+
+    }
+
+    /**
+     * Hook after create feature. @since ps v1.7.7
+     *
+     * @param array $params
+     */
+    public function actionAfterCreateFeatureFormHandler(array $params)
+    {
+        $this->save($params);
+    }
+
+    /**
+     * Hook after update feature. @since ps v1.7.7
+     *
+     * @param array $params
+     */
+    public function actionAfterUpdateFeatureFormHandler(array $params)
+    {
+        $this->save($params);
```

Comment:
No save method available

Reply:
Apparently i messed something with Feature and Feature value, ill fix it.